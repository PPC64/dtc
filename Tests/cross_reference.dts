/dts-v1/;

/{
	soc@deadbeef {
		baz: baz@0000 {
		};
	};

	foo {
		cross = &baz;
	};
};
