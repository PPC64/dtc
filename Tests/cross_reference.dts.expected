/dts-v1/;

/  {

	soc@deadbeef {

		baz@0000 {

		};
	};
	foo {

		cross = "/soc@deadbeef/baz@0000";
	};
};
